data/Abbas-Hussain-Muzammil.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Abbas Hussain Muzammil",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast | Blogger | Software Engineer",
+  "avatar": "https://github.com/Abbas-Hussain-Muzammil.png",
+// tags
```

Comment:
Comments are not valid json, this is causing the GitHub Action to fail

Reply:
Ok, removed them